nifi-nar-bundles/nifi-hive-bundle/nifi-hive-processors/src/test/java/org/apache/nifi/processors/hive/TestUpdateHiveTable.java

```diff
@@ -121,13 +120,10 @@
             new String[]{"Location:", "hdfs://mycluster:8020/warehouse/tablespace/managed/hive/_newTable", null}
     };
 
-    @Rule
-    public TemporaryFolder folder = new TemporaryFolder();
```

Comment:
This should be replaced with `@TempDir` and `Files.createTempDir()` should be removed in `testSetup()`.